engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/SafeHtmlValidatorTest.java

```diff
@@ -164,6 +164,20 @@ public void testAdditionalProtocols() {
 		assertNumberOfViolations( validator.validate( new Bar( "<div src='data:image/png;base64,100101' />" ) ), 1 );
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HV-817")
+	public void testPreserveRelativeLinks() throws Exception {
+		descriptor.setValue( "whitelistType", WhiteListType.RELAXED );
+		descriptor.setValue( "baseUri", "http://127.0.0.1" );
+
+		assertTrue( getSafeHtmlValidator().isValid( "<img src='/some/relative/url/image.png' />", null ) );
```

Comment:
> If the baseURI default value is set by default, the default behaviour of SafeHtml would change, since it currently does not allow relative paths.

Yeah, I'm wondering if we should consider it a feature or a bug. If, in the future, Jsoup checks the relative URL, I suppose it makes sense to keep it this way.

I was mostly wondering if people would really consider providing the baseURI in the annotation definition and if, maybe, we should just pass `http://localhost/` to Jsoup and consider relative links always valid from our POV.

I don't have a strong opinion about it though.